app/tests/e2e/test_issuer.py

```diff
@@ -111,7 +124,11 @@ async def test_store_credential(
 ):
     # TODO check for the correct response when state is credential_received
     # We can't complete this with auto accept enabled
-    time.sleep(5)
```

Comment:
Same with this tests. It doesn't make a lot of sense atm. bob sends an offer, so it makes sense it can't store the credential.